### PR TITLE
Improve wording and add defaults to TapHandler docs

### PR DESCRIPTION
**Pull request by jtalz**

### CODE REVIEW

File: docs/docs/handler-tap.md [hunk 1]
```diff
@@ -19,51 +19,51 @@ See [set of properties inherited from base handler class](handler-common.md#prop
...
 
 ### `maxDurationMs`
 
-Time expressed in milliseconds which defines how fast a finger has to be released after touch.
+Maximum time, expressed in milliseconds, that defines how fast a finger must be released after a touch. The default value is 500.
 
 ### `maxDelayMs`
 
-Time expressed in milliseconds which could pass before next tap if many taps are required. Default is 300ms. 
+Maximum time, expressed in milliseconds, that can pass before the next tap—if many taps are required. The default value is 500.
```
Comment (jakub-gonet) [inline docs/docs/handler-tap.md:30]:
```suggestion
Maximum time, expressed in milliseconds, that can pass before the next tap — if many taps are required. The default value is 500. 
```

File: docs/docs/handler-tap.md [hunk 2]
```diff
@@ -19,51 +19,51 @@ See [set of properties inherited from base handler class](handler-common.md#prop
...
 
 ### `numberOfTaps`
 
-Number of tap gestures required to [activate](state.md#active) the handler. Default is 1. 
+Number of tap gestures required to [activate](state.md#active) the handler. The default value is 1. 
 
 ### `maxDeltaX`
 
-When the finger travels the given distance expressed in points along X axis and handler hasn't yet [activated](state.md#active) it will fail recognizing the gesture.
+Maximum distance, expressed in points, that defines how far the finger is allowed to travel along the X axis during a tap gesture. If the finger travels further than the defined distance along the X axis and the handler hasn't yet [activated](state.md#active), it will fail to recognize the gesture.
```
Comment (jakub-gonet) [inline docs/docs/handler-tap.md:38]: Should we have trailing whitespace?

Comment (jtalz) [reply docs/docs/handler-tap.md]: removed it and a couple of others found (y) nice catch

**Review by jakub-gonet — APPROVED** [review]
Thanks! 🎉

**Pull request merged**